mrjob/options.py

```diff
@@ -526,9 +526,9 @@ def add_emr_opts(opt_group):
 
         opt_group.add_option(
             '--s3-endpoint', dest='s3_endpoint', default=None,
-            help=('Host to connect to when communicating with S3 (e.g.'
-                  ' s3-us-west-1.amazonaws.com). Default is to infer this from'
-                  ' region (see --aws-region).')),
+            help=('Force mrjob to connect to S3 on this endpoint (e.g.'
```

Comment:
Maybe this option should be dropped, or there should be a note here encouraging people not to use this, since mrjob connects to the right region automatically now.